content/en/getting_started/integrations.md

```diff
@@ -0,0 +1,140 @@
+---
+title: Getting started with Integrations
+kind: documentation
+aliases:
+  - /getting-started-integrations/
+  - /guides/getting-started-integrations/
+  - /guides/integrations/
+further_reading:
+- link: "https://learn.datadoghq.com/"
+  tag: "Learning Center"
+  text: "Introduction to Datadog"
+- link: "/integrations/"
+  tag: "Integrations"
+  text: "Datadog's full list of integrations"
+- link: "/graphing/"
+  tag: "Graphing"
+  text: "Now that you have your integrations set up, build graphs to visualize your data"
+- link: "/monitors/"
+  tag: "Alerts"
+  text: "Set up alerts on your data"
+---
+
+This is a guide for using integrations, for information about building a new integration, see the [Create a new integration][1] page.
+
+An integration, at the highest level, is when you assemble a unified system from units that are usually considered separately. At Datadog, you can use integrations to bring together all of the units of your infrastructure and gain insight into the unified system as a whole—you can see pieces individually and also how individual pieces are impacting the whole. Datadog's integrations use a Python class called `check` to define the typical metrics most people would want to monitor for that system.
+
+* Over 250 integrations are [officially listed][2], and supported in the [integrations core][3] (and more are always being added).
```

Comment:
```suggestion
* Over 250+ integrations are [officially listed][2], and supported in the [integrations core][3].
```

Reply:
The way i understand this sentence is that all integrations are living in integrations core, which is not true. 
We have indeed more than 250 integrations. All Agent related integrations officially supported by Datadog are in integrations-core, All Agent related integrations supported by the Datadog Community are in integrations-extras and all the other can be found in the DD integrations page: https://app.datadoghq.com/account/settings